### PR TITLE
[DO NOT MERGE] WIP of multiple assignees with differing completion

### PR DESCRIPTION
**Pull request by taylorz**

### CODE REVIEW

File: components/InfoDrawer.js [hunk 1]
```diff
@@ -100,11 +102,22 @@ const InfoDrawer = ({
...
-                    <Avatar
-                      size="mini"
-                      // TODO: can we get the assignee information for each task in the process serializer
-                      src={assignee && assignee.imageUrl}
-                    />
+                    <Stack spacing={2} direction="row">
+                      //TODO: map function to iterate through array of assignees
+                      <AvatarWrapper
+                        badgeContent={
+                          assignee && assignee.id === completedBy.id ? (
```
Comment (taylorz) [inline components/InfoDrawer.js:109]: If the assignee id matches the id of the user that completed the task, show the check badge

File: components/Task.js [hunk 2]
```diff
@@ -74,6 +75,7 @@ const Task = ({
   const [assignToastOpen, setAssignToastOpen] = useState(false);
   const [unassignToastOpen, setUnassignToastOpen] = useState(false);
   const [isDecided, setIsDecided] = useState(false);
+  const [completedBy, setCompletedBy] = useState(taskCompletedBy);
```
Comment (taylorz) [inline components/Task.js:78]: State to store who the task is completed by

Comment (keithtom) [reply components/Task.js:78]: I think this might need to be an array as well.

File: components/InfoDrawer.js (hunk 1 above)
Comment (keithtom) [reply components/InfoDrawer.js:109]: I'm imagining the assignee might have a completed attribute, but I need to flush this out in my backend work first.

File: components/Task.js [hunk 3]
```diff
@@ -386,8 +394,21 @@ const TaskDrawerActions = ({
...
+                    ? false
+                    : true
+                }
+                onClick={handleUncompleteTask}
+              >
+                <Typography bold>
+                  {completedBy && completedBy.id === currentUser.id
+                    ? "Mark incomplete"
+                    : `Completed by ${completedBy.firstName} ${completedBy.lastName}`}
+                </Typography>
```
[the remark this answers is not in the document]
Comment (keithtom) [reply components/Task.js:411]: I was thinking the backend might provide for each task an attribute `canComplete`, `canUncomplete` and you would show the buttons depending on these attributes.

To me the advantage of this is that it keeps the logic in 1 place in the backend vs duplicating it in the front end as well.

**Pull request merged**